File: c/120_pattern_12.c
#include<stdio.h>
int main() {
    int n;
    printf("Enter n :- ");
    scanf("%d",&n);
    for(int i=0;i<n;i++) {
        for(int j=0;j<n;j++) {
            if(i == 0 || i == n-1) {
                printf("*");
            }
            else {
                if(n/2 == j) printf("*");
                else printf(" ");
            }
        }
        printf("\n");
    }
}

/*

Output :-
Enter n :- 5
*****
  *
  *
  *
*****

*/

File: python/code/120_pattern12.py
n = int(input("Enter n :- "))
for i in range(n):
    for j in range(n):
        if(i == 0 or i == n-1):
            print("*", end='')
        else:
            if(n//2 == j):
                print("*", end='')
            else: 
                print(" ", end='')
    print()
    
# Output :-
# Enter n :- 5
# *****
#   *
#   *
#   *
# *****